src/Concerns/CapturesState.php

```diff
@@ -458,9 +458,9 @@ public function attachMiddlewareToRoute(Route $route): void
     /**
      * @internal
      */
-    public function waitForJob(): void
+    public function waitForExecution(): void
     {
-        $this->waitingForJob = true;
+        $this->sample(0);
```

Comment:
🍸  Because it isn't clear what the `0` is for, I always prefer to include the named parameter. I designed the API with the intention of making it readable, e.g., sample rate of zero.

```suggestion
        $this->sample(rate: 0);
```

Reply:
I updated it to use the `dontSample` method instead. What do you think? [1c3d8d4](https://github.com/laravel/nightwatch/pull/265/commits/1c3d8d403e2af7a5fdb1632efb5865d5a0ab71e2)

Reply:
Perfection!